File: code/basics/range-operator.scala
val names = Array(
  "Anna", "Alex", "Brian", "Jack")
val count = names.length
for (i <- 0 until count) {
  println(
    s"Person ${i + 1} is ${names(i)}")
}
// Person 1 is Anna
// Person 2 is Alex
// Person 3 is Brian
// Person 4 is Jack

File: code/basics/range-operator.go
names := [4]string{
  "Anna", "Alex", "Brian", "Jack"}
for i, value := range(names) {
  fmt.Printf(
    "Person %v is %v\n", (i + 1), value)
}
// Person 1 is Anna
// Person 2 is Alex
// Person 3 is Brian
// Person 4 is Jack